test/test_plugins.cpp

```diff
@@ -27,7 +27,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <pluginlib/class_list_macros.h>
+#include <pluginlib/class_list_macros.hpp>
 #include "test_base.h"  // NOLINT
 #include "test_plugins.h"  // NOLINT
```

Comment:
Should these include statements also be updated to use `#include "./HEADER_NAME"` rather than being explicitly ignored by cpplint?

Reply:
Maybe, I'd say out-of-scope for this pr.

Reply:
fair enough :+1: